frontend/public/module/k8s/service-catalog.ts

```diff
@@ -7,6 +7,9 @@ import { K8sResourceKind } from '../../module/k8s';
 export const serviceClassDisplayName = (serviceClass: K8sResourceKind): string =>
   _.get(serviceClass, 'spec.externalMetadata.displayName') || _.get(serviceClass, 'spec.externalName');
 
+export const servicePlanDisplayName = (servicePlan: K8sResourceKind): string =>
+  _.get(servicePlan, 'spec.externalMetadata.displayName') || _.get(servicePlan, 'spec.externalName') || _.get(servicePlan, 'metadata.name');
```

Comment:
`spec.externalName` will always be set, so no need to fallback to `metadata.name`